release-notes/swan-lake-beta2-release-note.md

```diff
@@ -68,6 +68,11 @@ name="John"
 age=10
 ```
 
+#### Test Framework
```

Comment:
This seems to have been added under the "Language Updates" section. Shouldn't this come under https://github.com/ballerina-platform/ballerina-release/blob/master/release-notes/swan-lake-beta2-release-note.md#developer-tools-updates?

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-release/pull/650